src/serde/deserialize_utils.rs

```diff
@@ -159,7 +159,14 @@ pub fn parse_value(input: &str) -> IResult<&str, ValueAddress> {
         opt(offset),
     ))(input)?;
 
-    let (_, (_, type_)) = tuple((tag(", "), take_till(|c: char| c == '*')))(second_arg)?;
+    let (reference_count, (_, struct_)) =
```

Comment:
I'd say rename this variable. `reference_count` sounds misleading as it may be confused with the memory management technique.
```suggestion
    let (indirection_level, (_, struct_)) =
```

Reply:
Nice, sounds better!
Done